### PR TITLE
Remove spaces from folder names

### PR DESCRIPTION
**Pull request by caiohsramos**
- Rename folders to keep the kebab-case standard

### CODE REVIEW

File: content/docs/faq/_index.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ As for non-moderation ways you can keep things running: Contribute to discussion
 ## What else should I know?
 We *always* take feedback. While we can’t promise any *changes*, you can provide thoughts and comments on just about anything on the site in [Beehaw Support](https://beehaw.org/c/support), or on [Discord] or [Matrix] (where we maintain real-time chat). We try to get a sense of what the community wants and needs all the time, so we’ll frequently be asking for input from you to help inform our decisions.
 
-- [Downvotes are disabled on this instance](https://docs.beehaw.org/philosophy/why-downvotes-are-disabled/) and will probably always be. we find them unproductive. 
-- [Communities cannot be made by users on here](https://docs.beehaw.org/docs/philosophy/beehaw-is-a-community/). we do take feedback on what gets made and when to make communities, as a part of the above point. you can always find a running list of communities we have [here](https://beehaw.org/communities), or under the Communities button on desktop.
+- [Downvotes are disabled on this instance]({{< ref "/docs/core-functions/why-downvotes-are-disabled" >}}) and will probably always be. we find them unproductive.
```
Comment (lionirdeadman) [inline content/docs/faq/_index.md]: Good catch

File: content/docs/core-functions/_index.md
Comment (lionirdeadman) [inline content/docs/core-functions/_index.md:1]: I think all of these changes while good should probably be made in another PR to avoid messing with the commit history..

**Review by lionirdeadman — CHANGES_REQUESTED** [review]

File: content/docs/core-functions/_index.md
Comment (caiohsramos) [reply content/docs/core-functions/_index.md:1]: I can edit my branch and break into two commits, then you just need to "Rebase and merge" in Github. What do you think?

Comment (lionirdeadman) [reply content/docs/core-functions/_index.md:1]: I am not good enough with git to know what Rebase and merge does to be quite honest. My primary worry is to keep git history more readeable and avoid git conflicts though I guess

Comment (caiohsramos) [reply content/docs/core-functions/_index.md:1]: No worries, I'll keep this one as the folders names change and create a new one to fix broken links
